bin/check-examples.sh

```diff
@@ -8,64 +8,65 @@
 set -e
 
 THIS_DIR=$(dirname $0)
```

Comment:
The `$0` needs to be quoted, otherwise it fails if the path contains a space.

```suggestion
THIS_DIR="$(dirname "$0")"
```